### ISSUE TITLE
Alt text character limit is too low

### ISSUE THREAD

**Issue by ymau** [pixelfed/mobile-app#78]
In the Android mobile app the alt text is restricted to 140 characters which can be too short for a good description. IIRC the web version allows alt texts to consist of up to 2000 characters. I think 500 characters should be the lower limit for an expressive description.

**Comment by intentionally-left-nil** [comment]
Having the alt text > than the text cap makes sense to me
